core/controlplane/config/templates/cluster.yaml

```diff
@@ -397,7 +405,7 @@ worker:
 #        # IAM role to grant the Spot fleet permission to bid on, launch, and terminate instances on your behalf
 #        # See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-fleet-requests.html#spot-fleet-prerequisites
 #        #
-#        # Defaults to "arn:aws:iam::youraccountid:role/aws-ec2-spot-fleet-role" assuming you've arrived "Spot Requests" in EC2 Dashboard
+#        # Defaults to "arn:aws:iam::youraccountid:role/aws-ec2-spot-fleet-tagging-role" assuming you've arrived "Spot Requests" in EC2 Dashboard
```

Comment:
Seems like part of https://github.com/kubernetes-incubator/kube-aws/pull/1024/ slipped in here?

Reply:
Yes, you are correct. Sorry but I had no time to test this in isolation, as noted in `Depends on #1024` in the PR description!